### DISCUSSION TITLE
how to avoid blocked valves?

### DISCUSSION THREAD

**Discussion by harvey637** [Lenti84/VdMot_Controller#88] [General]
Hi @ Gargano and @ Lenti84,
now running vdmot_controller for about 2 month, mostly successfull. I am on beta 1.2 since some time, working fine by now, not yet tested email...

In this two month I had 3 "blocked" valves, I am using the original ELV-vdmot valves with high/low of 1.6 .
2 valves where "out" so I could get them "in" again with extra connection to the valve and a batterypack.
But one valve is destroyed, as the "nose", the little plastic that prevents the pushing mechanism from turning, is broken so the valve is not working anymore. Sad story, I never had this with the original falmot-c12.

So I have to buy new vdmot, this will take some time (and money too :-( ).

What can I do to prevent this from happening again?
- lower the high/low values e.g. to 1.4?
- or only the lower to 1.4, leaving the higher at 1.6, as the "overturning" to the outside does not destroy the vdmot, as this "pin" simply moves outside the "tunnel". 
- some more checks inside the firmware?
- as the blocked valves "outside" jump out of the adapter (looks tilted, after recalibrating I push the vdmot back) - should  I connect the vdmot to the adapter strongly with a screw?
- this will not prevent the "inside" block, as this sadly destroyed the vdmot.

Other than this really bad story the vdmot_controller run near perfect, with little observations:
- with window sensors I detect open windows/doors and lower the room temperature (injected into tValue), after closing the door the valve opens, even if the temperature is already perfect sufficient and no heating looks neccessary (see screenshot). With my door to the terrace (which is opend and closed very often) this is a bit confusing, (very very short openings do not show vdmot movements, as the sampling rate of valve percent is quit low compared to the door open/close, but the 19:45 shows the problem perfectly)
- any info on the display which was mentioned in the docs?
- readonly status page without button to manually override valve position AND WITHOUT login/password would be nice
- maybe some more attractive optics on this page, eg. (simple) bar chart, nicer and less technical look, open/close/idle as arrows/icons ...

![grafik](https://user-images.githubusercontent.com/10156426/235341816-39f39642-a77c-46c1-962f-7041f670154f.png)

Looking forward hearing from you
Harvey

**Comment by SurfGargano** [comment]
For blocking valves pls reduce high/low value. In my environment is a value of 1.5 good. But 1.4 should also working. For the other issues I can answer when I am back from holiday.
Thx for testing.

**Comment by harvey637** [comment]
Happy Holiday and good surf!
With 1.5 I did not observe blocked valves. Also I managed to "repair" the broken valve: I drilled a small hole and put a small screw into that hole, than I rasped (is this crorreect?) the screw so no more turns on it: works since a week. Maybe the friction between the iron of the screw an the plastic of the housing will be a future problem. In my opinion the valve is badly constructed: this very small pin could be avoided if the design is a square sliding hole, it looks like enough space, but ELV/eq3 is shure not interested in that change.

One other observation is a strange behaviour:
![grafik](https://github.com/Lenti84/VdMot_Controller/assets/10156426/22ab6caf-1ba0-42ec-bc87-417eac53eae0)

Only saw it twice, I am not shure weather this is simple bad sampling of the valve percent (via api call) or possibly a recalibration due to high/low seettings? It is just an observation, no manual handling was performed.

have a good time and hear from you anytime after holiday!
It is a pleasure to talk to you
ciao
Harvey

**Comment by SurfGargano** [comment]
Thx. Actually we are in napoli  coming from rome and florence. As it raining cats and dogs, we will go to vieste on friday, starting surfing, bycicle and sup.Am 17.05.23, 20:51 schrieb harvey637 ***@***.***>:

   
  Happy Holiday and good surf! With 1.5 I did not observe blocked valves. Also I managed to "repair" the broken valve: I drilled a small hole and put a small screw into that hole, than I rasped (is this crorreect?) the screw so no more turns on it: works since a week. Maybe the friction between the iron of the screw an the plastic of the housing will be a future problem. In my opinion the valve is badly constructed: this very small pin could be avoided if the design is a square sliding hole, it looks like enough space, but ELV/eq3 is shure not interested in that change. 
  One other observation is a strange behaviour:  
  Only saw it twice, I am not shure weather this is simple bad sampling of the valve percent (via api call) or possibly a recalibration due to high/low seettings? It is just an observation, no manual handling was performed. 
  have a good time and hear from you anytime after holiday! It is a pleasure to talk to you ciao Harvey

**Reply by harvey637** [reply]
saw it in the news, much too dry or much too mauch rain .... this is why I am interested in optimizing my home for less energy (with heatpump and PV) ....
Hope You get same fine days in your holiday. ciao
Harvey
